### PR TITLE
Update instance_analytics for 50 major release

### PR DESCRIPTION
**Pull request by luizarakaki**
Serialized using 50.22

### CODE REVIEW

File: resources/instance_analytics/collections/vG58R8k-QddHWA7_47umn_metabase_analytics/dashboards/BHyad8ZHCbeiBZpQxDwsP_content_with_cobwebs.yaml [hunk 1]
```diff
@@ -28,64 +27,5 @@ serdes/meta:
 - id: BHyad8ZHCbeiBZpQxDwsP
   label: content_with_cobwebs
   model: Dashboard
-dashcards:
```
Comment (luizarakaki) [inline resources/instance_analytics/collections/vG58R8k-QddHWA7_47umn_metabase_analytics/dashboards/BHyad8ZHCbeiBZpQxDwsP_content_with_cobwebs.yaml:31]: not sure what is going on here cc @piranha @crisptrutski

Comment (piranha) [reply resources/instance_analytics/collections/vG58R8k-QddHWA7_47umn_metabase_analytics/dashboards/BHyad8ZHCbeiBZpQxDwsP_content_with_cobwebs.yaml:31]: uhhh this is very concerning; are you sure it's 50.22? I've just checked locally and I get those dashcards...

**Comment by crisptrutski** [comment]
It seems that the audit instance is running version `80813cb`, which is an old version of master which predated the bug-bashing involved in the refactor. It seems that this issue was fixed before the changes as a whole were backported, so I think we're in the clear.

**Comment by luizarakaki** [comment]
> It seems that the audit instance is running version `80813cb`, which is an old version of master which predated the bug-bashing involved in the refactor. It seems that this issue was fixed before the changes as a whole were backported, so I think we're in the clear.

Yeah, my mistake. I used to serialize using a local jar, but now we do it using the API and for some reason the instance was outdated. I restarted the instance and it is now following `master` again.

Yamls look good now

**Review by crisptrutski — APPROVED** [review]
LGTM

**Comment by piranha** [comment]
@luizarakaki I wonder if "fix another random fk" is done by hands or by fixing it on stats?

**Comment by luizarakaki** [comment]
> @luizarakaki I wonder if "fix another random fk" is done by hands or by fixing it on stats?

I'm fixing these issues in the source instance: audit-dev. We don't use Stats to develop audit content

**Review by noahmoss — APPROVED** [review]

**Pull request merged**